discovery_engine/lib/src/domain/assets/asset.dart

```diff
@@ -65,6 +65,8 @@ class Asset {
 enum AssetType {
   smbertVocab,
   smbertModel,
+  webWorkerScript,
+  wasmScript,
```

Comment:
I think we can remove them for now. The issue is that we might have different `webWorkerScript` and `wasmScript` assets (depending on the feature set that the browser supports). I hope that we will find a better solution then we have in `xayn_ai` now because it was quite hacky.